lib/app/components/pages/search_select_genre/controllers/search_select_genre_controller.dart

```diff
@@ -17,13 +17,14 @@ class SearchSelectGenreController extends GetxController {
   late final List<Map<String, dynamic>> _genreMapList;
   final RxList<SearchGenreInfo> genreForSelect = <SearchGenreInfo>[].obs;
   String _selectGenreId = "";
+  String get genreId => _selectGenreId;
 
   @override
   void onInit() async{
```

Comment:
```suggestion
  Future<void> onInit() async{
```
こっちも`Future<void>`にしておきましょうか

Reply:
@king-kazu39 
ありがとうございます！
修正しました！